models/alignn_ff/metadata.yml

```diff
@@ -0,0 +1,42 @@
+model_name: ALIGNN FF
+model_version: 2023.07.01
+matbench_discovery_version: 1.0
+date_added: "2023-07-11"
+date_published: "2022-09-16"
+authors:
+  - name: Kamal Choudhary
+    affiliation: National Institute of Standards and Technology
+    email: kamal.choudhary@nist.gov
+    orcid: https://orcid.org/0000-0001-9737-8074
+  - name: Brian DeCost
+    affiliation: National Institute of Standards and Technology
+    orcid: https://orcid.org/0000-0002-3459-5888
+  - name: Lily Major
+    affiliation: Aberystwyth University, UK
+    orcid: https://orcid.org/0000-0002-5783-8432
+  - name: Keith Butler
+    affiliation: Rutherford Appleton Laboratory, UK
+    orcid: https://orcid.org/0000-0001-5432-5597
+  - name: Jeyan Thiyagalingam
+    affiliation: Rutherford Appleton Laboratory, UK
+    orcid: https://orcid.org/0000-0002-2167-1343
+  - name: Francesca Tavazza
+    affiliation: National Institute of Standards and Technology
+    orcid: https://orcid.org/0000-0002-5602-180X
+  - name: Philipp Benner
+    affiliation: Bundesanstalt für Materialforschung und -prüfung BAM
+    orcid: https://orcid.org/0000-0002-0912-8137
+    github: https://github.com/pbenner
+repo: https://github.com/usnistgov/alignn
+url: https://jarvis.nist.gov/jalignn
+doi: https://doi.org/10.1039/D2DD00096B
+preprint: https://arxiv.org/abs/2209.05554
+requirements:
+  ase: 3.22.0
```

Comment:
@pbenner Could you check if these version numbers are correct (i.e. match the ones you were using for this submission).

Reply:
Yes, the versions are correct, except for ase -> 3.22.1